tests/trustyai/conftest.py

```diff
@@ -86,3 +99,144 @@ def minio_data_connection(
         },
     ) as minio_secret:
         yield minio_secret
+
+
+@pytest.fixture(scope="class")
+def db_credentials(admin_client: DynamicClient, model_namespace: Namespace) -> Secret:
+    with Secret(
+        client=admin_client,
+        name="db-credentials",
+        namespace=model_namespace.name,
+        string_data={
+            "databaseKind": MARIADB,
+            "databaseName": "trustyai_database",
+            "databaseUsername": QUARKUS,
+            "databasePassword": QUARKUS,
+            "databaseService": MARIADB,
+            "databasePort": "3306",
+            "databaseGeneration": "update",
+        },
+    ) as db_credentials:
+        yield db_credentials
+
+
+@pytest.fixture(scope="session")
+def mariadb_operator(admin_client: DynamicClient) -> None:
```

Comment:
```suggestion
def installedmariadb_operator(admin_client: DynamicClient) -> None:
```
to explain what the fixture is about